docs/extensibility/debugger/reference/idebugcodecontext2-getdocumentcontext.md

```diff
@@ -42,7 +42,9 @@ int GetDocumentContext( 
 
 ## Remarks
  Generally, the document context can be thought of as a position in a source file while the code context is a position of a code instruction in an execution stream.
+ 
+ If no document context is available, returning `S_FALSE` is the desired behavior.
```

Comment:
Doesn't this note belong under 'Return Value'?

Reply:
I moved this comment and reworded it a bit.